<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgInvestments</class>
 <widget class="QWidget" name="wdgInvestments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>525</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lbl">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Investments list</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkInactivas">
         <property name="text">
          <string>Show inactive investments</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="myQTableWidget" name="tblInvestments">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Investment</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Last datetime</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Last value</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Daily difference</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>% Intraday</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Balance</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Pending</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>% Invested</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>% Selling point</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="wdgIBM" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblTotal">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionInvestmentAdd">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>New investment</string>
   </property>
   <property name="toolTip">
    <string>New investment</string>
   </property>
  </action>
  <action name="actionInvestmentReport">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/bundle.png</normaloff>:/xulpymoney/bundle.png</iconset>
   </property>
   <property name="text">
    <string>Investment report</string>
   </property>
   <property name="toolTip">
    <string>Investment report</string>
   </property>
  </action>
  <action name="actionSortTPCDiario">
   <property name="icon">
    <iconset>
     <normaloff>:/xulpmoney/document-preview.png</normaloff>:/xulpmoney/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>% Daily</string>
   </property>
   <property name="toolTip">
    <string>% Daily</string>
   </property>
  </action>
  <action name="actionSortTPCVenta">
   <property name="icon">
    <iconset>
     <normaloff>:/xulpmoney/document-preview.png</normaloff>:/xulpmoney/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>% Selling point</string>
   </property>
   <property name="toolTip">
    <string>% Selling point</string>
   </property>
  </action>
  <action name="actionSortTPC">
   <property name="icon">
    <iconset>
     <normaloff>:/xulpmoney/document-preview.png</normaloff>:/xulpmoney/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>% Invested</string>
   </property>
   <property name="toolTip">
    <string>% Invested</string>
   </property>
  </action>
  <action name="actionSortName">
   <property name="icon">
    <iconset>
     <normaloff>:/xulpmoney/document-preview.png</normaloff>:/xulpmoney/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="toolTip">
    <string>Name</string>
   </property>
  </action>
  <action name="actionSortHour">
   <property name="icon">
    <iconset>
     <normaloff>:/xulpmoney/document-preview.png</normaloff>:/xulpmoney/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
   <property name="toolTip">
    <string>Time</string>
   </property>
  </action>
  <action name="actionActive">
   <property name="text">
    <string>Is it active?</string>
   </property>
   <property name="toolTip">
    <string>Is it active?</string>
   </property>
  </action>
  <action name="actionProduct">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/books.png</normaloff>:/xulpymoney/books.png</iconset>
   </property>
   <property name="text">
    <string>Product report</string>
   </property>
   <property name="toolTip">
    <string>Product report</string>
   </property>
  </action>
  <action name="actionProductPrice">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>Add product price</string>
   </property>
   <property name="toolTip">
    <string>Add product price</string>
   </property>
  </action>
  <action name="actionInvestmentDelete">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/list-remove.png</normaloff>:/xulpymoney/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete investment</string>
   </property>
   <property name="toolTip">
    <string>Delete investment</string>
   </property>
  </action>
  <action name="actionProductPriceLastRemove">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/list-remove.png</normaloff>:/xulpymoney/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove last product price</string>
   </property>
   <property name="toolTip">
    <string>Remove last product price</string>
   </property>
  </action>
  <action name="actionSameProduct">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/bundle.png</normaloff>:/xulpymoney/bundle.png</iconset>
   </property>
   <property name="text">
    <string>Same product Investments merging current operations</string>
   </property>
   <property name="toolTip">
    <string>Same product Investments merging current operations</string>
   </property>
  </action>
  <action name="actionSameProductFIFO">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/bundle.png</normaloff>:/xulpymoney/bundle.png</iconset>
   </property>
   <property name="text">
    <string>Same product Investments merging operations</string>
   </property>
   <property name="toolTip">
    <string>Same product Investments merging operations</string>
   </property>
  </action>
  <action name="actionProductUpdate">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/transfer.png</normaloff>:/xulpymoney/transfer.png</iconset>
   </property>
   <property name="text">
    <string>Update product</string>
   </property>
   <property name="toolTip">
    <string>Update product</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">myqtablewidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
